src/header/nav.md

```diff
@@ -3,7 +3,7 @@ template: header-nav
 nav_items:
   - label: Join
     url: /join
-  - label: Stake
+  - label: Mint tBTC
     url: /stake
```

Comment:
Shouldn't we change the url to `https://threshold.network/earn/btc` ?

Reply:
Do we want to keep a link to `stake` page? If yes we should add the `Mint tBTC` link as a new nav item in `nav_items`.

Reply:
What I would actually like to do is replace this button here on the far right with `Mint TBTC` instead of `Launch`. I didn't know how to accomplish this with the CMS.

<img width="616" alt="Screenshot 2023-01-27 at 1 43 47 PM" src="https://user-images.githubusercontent.com/57226633/215090027-6c7f9eb3-a550-4dc2-b170-acc9cc247f44.png">

Then, we would also have a new navigation item in nav_items that is `tBTC` that re-directs to the Threshold informational page about tBTC.

This would be the URL for now for both: https://threshold.network/earn/btc/
But, ultimately at launch the `MINT TBTC` button should link to the Threshold dapp.

And yeah, we likely _should_ maintain `Stake` in the navigation. Upon further reflection, we do reference that page in other parts of the website so we would need to replace it everywhere.